### PR TITLE
Edit for brevity

### PR DESCRIPTION
**Pull request by mangstadt**
Sorry, I couldn't myself. It's way too wordy IMO. My experience with interacting with people over email and other electronic written media is that shorter is better! Anyway, have a good day! :)

### CODE REVIEW

**Review by mpagels — COMMENTED** [review]
Hi @mangstadt, I like your changes very much. But I also like the original message very much. Maybe instead of changing the original message completely, it would be a good idea to add your message as a "shorter" version? That way, people can decide which version they want to use? Just an idea.

**Comment by Kyu** [comment]
I like this a lot, commenting to save
